.circleci/config.yml

```diff
@@ -78,6 +78,9 @@ jobs:
           cd ..
           git clone git@github.com:lumigo-io/larn.git
     - checkout
+    - run: apt-get update
+    - run: apt-get install jq
+    - run: npm install -g aws-cdk@1.125.0
```

Comment:
@moshe-shaham-lumigo why do you need CDK?

Reply:
because all integration tests now work with the new edge

Reply:
So all services will need this as part of their circle config?

Reply:
yes
everyone that uses the orb already have it

Reply:
ok, I would recommend updating our base docker at https://github.com/lumigo-io/utils/blob/master/deployment/ci-docker/Dockerfile it will improve our CI running time